server/src/dummyDb/db.js

```diff
@@ -57,5 +57,38 @@ export default {
       status: 'dormant',
       balance: 2364756.00
     }
+  ],
+  transactions: [
+    {
+      id: 1,
+      createdOn: 'Thursday, May 30, 2019 5:11 PM',
+      transactypeType: 'credit',
+      accountNumber: 9987456386,
+      cashier: 2,
+      amount: 2000.00,
+      oldBalance: 4000.00,
+      newBalance: 6000.00
+    },
+    {
+      id: 2,
+      createdOn: 'Thursday, May 30, 2019 5:11 PM',
+      transactypeType: 'debit',
+      accountNumber: 1123986435,
+      cashier: 3,
+      amount: 6000.00,
+      oldBalance: 10000.00,
+      newBalance: 4000.00
+    },
+    {
+      id: 3,
+      createdOn: 'Thursday, May 30, 2019 5:11 PM',
```

Comment:
Are you sure this is what, `moment().format()` evaluates to?

Reply:
done...